x-pack/plugins/security_solution/public/management/routes.tsx

```diff
@@ -6,11 +6,22 @@
  */
 
 import React from 'react';
-import { MANAGEMENT_PATH } from '../../common/constants';
+import { TrackApplicationView } from '@kbn/usage-collection-plugin/public';
+import { MANAGEMENT_PATH, MANAGE_PATH } from '../../common/constants';
+import { SecurityPageName } from '../app/types';
 import { ManagementContainer } from './pages';
 import type { SecuritySubPluginRoutes } from '../app/types';
 import { CurrentLicense } from '../common/components/current_license';
 import { PluginTemplateWrapper } from '../common/components/plugin_template_wrapper';
+import { ManageLandingPage } from './pages/landing';
+
+const ManagementLanding = () => (
```

Comment:
We always try to use memoization as guideline in our code based and not really think about the conditions where it is not needed. Not every developer understands the conditions where memoization might not be needed and since there really is no drawback from using it, it's just easier and less burden on devs.

Reply:
Okay, no problem. I added the memo on both routes' `component`